DoxygenHelp.md

```diff
@@ -22,27 +22,37 @@ If DOXYGEN_GENERATE_DOCUMENTATION is selected in the CMake GUI, CMake will autom
 https://github.com/tumcms/Open-Infra-Platform/blob/development/CMakeLists.txt
 
 In the CMake GUI, the following options can be selected:
-- DOXYGEN_OPTIONAL_COMMENTED_ONLY: If this is not selected, doxygen will assume that all entities are documented, even if there is no documentation available. 
-- DOXYGEN_OPTIONAL_INCLUDE_INTERNAL: If selected, this will include all comments preceded by "\internal" (comments intended for developers only).
-- DOXYGEN_OPTIONAL_INCLUDE_EARLYBINDING: If selected (... and already built), this will include the selected IFC schemas in the documentation. 
+- DOXYGEN_OPTIONAL_INCLUDE_COMMENTED_ONLY: If selected, only documentation for explicitly commented entities will be generated. Otherwise, doxygen will assume that all entities are documented, even if there is no documentation available. 
+- DOXYGEN_OPTIONAL_INCLUDE_INTERNAL: If selected, this will include all comments preceded by "\internal". These are comments intended for developers only. So if the generated documentation is intended for external users, this option should not be selected.
+- DOXYGEN_OPTIONAL_INCLUDE_EARLYBINDING: If selected (... and already built), this will include the selected IFC schemas in the documentation. Otherwise, the schemas will not be included in the documentation.
+- DOXYGEN_OPTIONAL_AUTO_OPEN_DOCUMENTATION: If selected, the index.html page of the documentation will automatically open. If not selected, you will have to find and open the generated documentation manually (see section below on building the documentation). 

```

Comment:
Do you mean just stating what should be default or actually setting default values for the options? I tried e.g Option(DOXYGEN_GENERATE_DOCUMENTATION ON) and all others as OFF, then it still creates the target and configures everything although the box is not ticked in the CMake GUI - this might be confusing. Is there a way to pretick the box somehow?

Reply:
Well, both in some way. The defaults as set by the CMakeLists should be documented. Which are set as ON, this is our decision. I would set generate: on, internal: off, commented only: on, early binding: off.

I think to preset an option in CMake, you provide the OPTION( ) with a value. Note: you need to delete your cache to delete the options set previously.

Reply:
Resolved: https://github.com/tumcms/Open-Infra-Platform/pull/104/commits/eaa6f4468255729f59d13884a0ef8ac46621722d 

and 

https://github.com/tumcms/Open-Infra-Platform/pull/104/commits/3fcfd00ee796c8efc36270bde925642e31c6bffc